lib/Jsrt/JsrtCommonExports.inc

```diff
@@ -5,6 +5,7 @@
     JsAddRef
     JsRelease
     JsCreateContext
+    JsGetContextUrl
```

Comment:
Yes new API needs to be reviewed.

In this case, I don't think a new API is necessary. ch should be able to keep track of the script file name it is running, and get the filename and directory.
It is not the "current" script file, but it should be enough for what you need.